Samples/Swift/DaysUntilBirthday/Shared/ViewModels/VerifiedAgeViewModel.swift

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import GoogleSignIn
+
+#if os(iOS)
+
+final class VerifiedAgeViewModel: ObservableObject {
+  /// The user's account verification status.
+  /// - note: This will publish updates when its value changes.
+  @Published var verificationState: VerificationState
+
+  private var loader: VerificationLoader {
```

Comment:
This will make a new `loader` every time it is called. Is that necessary? My sense is that you can instantiate once in the initializer. 

I know that `AuthenticationViewModel` does this, but that could be a mistake or for some unrelated reason; so, I figured I'd ask. :)

Reply:
I guess part of the issue is that `GoogleSignInAuthenticator` and this loader pass themselves into the respective initializers. That means we can't create instances of these loaders in the view model initializers. I wonder if you could make these `lazy var`s? E.g.,

```
private lazy var loader: VerificationLoader = {
  return VerificationLoader(verifiedViewAgeMode: self)
}()
```

Reply:
Looks like making it a `lazy var` works. Did we want this change on `GoogleSignInAuthenticator` as well?

Reply:
Let's leave the other one as is for now. We can make the change there if there is a need.